### PR TITLE
NEP-514: Reducing the number of Block Producer Seats in `testnet`

### PR DESCRIPTION
**Pull request by nikurt**

### CODE REVIEW

**Review by gmilescu — APPROVED** [review]
The NEP looks good. Adapting the testnet size to the existing number of nodes simplifies testing.

**Comment by nikurt** [comment]
ping @frol @ori-near

**Comment by nikurt** [comment]
Begging @frol and @ori-near for a review

File: neps/nep-0514.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+#### Concerns
+
+> Template for Subject Matter Experts review for this version:
+> Status: New | Ongoing | Resolved
+
+|   # | Concern | Resolution | Status |
+| --: | :------ | :--------- | -----: |
+|   1 |         |            |        |
+|   2 |         |            |        |
```
Comment (frol) [inline neps/nep-0514.md:169]: Given that Benefits and Concerns are already summarized in "Consequences" section, the benefits and concerns may be skipped here, but we still need to capture how the decision has been made. Here is an example:

```suggestion
### 1.0.0 - Initial Version

The Protocol Working Group members approved this NEP on MMM DD, 2023
```

**Review by frol — CHANGES_REQUESTED** [review]
Thank you @nikurt for submitting this NEP, and sorry for the delay.

As a moderator, I reviewed this NEP and it meets [the proposed template guidelines](https://github.com/near/NEPs/blob/master/nep-0000-template.md).

Since I see that the request comes from nearcore team and is approved by subject matter experts (@gmilescu and @wacban [on reference implementation PR]), I believe we should have moved to [the REVIEW stage](https://github.com/near/NEPs/blob/master/neps/nep-0001.md#nep-lifecycle) with @near/wg-protocol working group members reviewing and voting on it, BUT it seems that the PR to nearcore seems to be already merged, so I guess this is just a formality now. Overall, I don't want to make this process unnecessarily hard, so I am fine to rely on just a single vote from Protocol WG (@bowenwang1996, do you want to cast your Protocol WG vote here?)

**Comment by nikurt** [comment]
@bowenwang1996 @mfornet @birchmd Please take a look

**Review by bowenwang1996 — APPROVED** [review]
Looks good to me

**Review by frol — APPROVED** [review]
Based on my previous comment and getting LGTM from the Protocol WG, I approve and merge this PR

**Pull request merged**